pkg/testutil/testutil_windows.go

```diff
@@ -43,7 +43,9 @@ const (
 	NginxAlpineImage            = "registry.k8s.io/e2e-test-images/nginx:1.14-2"
 	NginxAlpineIndexHTMLSnippet = "<title>Welcome to nginx!</title>"
 
-	GolangImage = "fixme-test-using-this-image-is-disabled-on-windows"
+	GolangImage  = "fixme-test-using-this-image-is-disabled-on-windows"
```

Comment:
As we remove more of platform-specific test files and replace with platform-dependent `t.Skip`, we need a few placeholders so that we compile.

The point of this (as exposed ealier), is to make platform-specific skip/unskip easier than it is today.